crates/rethnet/src/config.rs

```diff
@@ -34,6 +34,7 @@ pub const DEFAULT_PRIVATE_KEYS: [&str; 20] = [
 #[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
 pub struct ConfigFile {
     // TODO: expand this per https://github.com/NomicFoundation/rethnet/issues/111
+    pub allow_blocks_with_same_timestamp: Option<bool>,
```

Comment:
```suggestion
    #[serde(default)]
    pub allow_blocks_with_same_timestamp: bool,
```

Reply:
In [the other PR](https://github.com/NomicFoundation/hardhat/pull/4221) I settled on the pattern of just doing `#[serde(default)]` at the struct level rather than at the field level, since many of these fields need specific values as their defaults rather than just the default value for their type. For that reason, I'd prefer to leave out the `#[serde(default)]` here, instead specifying the default value in `ConfigFile::default()`.  However, the `Option` removal part of your suggestion I did take, when I [merged the `rethnet/142-instantiate-blockchain-mempool-and-miner-in-server` branch into this branch](https://github.com/NomicFoundation/hardhat/pull/4227/commits/a44f2f68ddd9f0cced4c1309322746cbcefe627b).